buildchain/buildchain/constants.py

```diff
@@ -91,8 +91,8 @@ def git_ref() -> Optional[str]:
 
     try:
         ref : bytes = subprocess.check_output([
-            config.GIT, 'describe', '--always', '--long', '--tags',
-                '--dirty',
+            config.ExtCommand.GIT.value, 'describe', '--always', '--long',
```

Comment:
about commit message: the reference to the issue is usually put at the end of the commit, not in the title.
Also, we usually prefix the title with the component/module we are working on (in that case `build` or `buildchain`).

e.g.:

```
build: add task dep on binaries checking

For every task that rely on an external command, we add a dependence to the task that check the existence of the required binary.
```